cmake/Common.cmake

```diff
@@ -0,0 +1,116 @@
+# Definitions:
+
+if (NOT MEASUREMENTKIT_ROOT)
+    set (MEASUREMENTKIT_ROOT "${CMAKE_SOURCE_DIR}")
+endif (NOT MEASUREMENTKIT_ROOT)
```

Comment:
This is added. If allows somebody importing this cmake file to override the `MEASUREMENTKIT_ROOT` with their own path to where measurement-kit lives. This will be clear why below.